src/js/serial_backend.js

```diff
@@ -688,8 +691,15 @@ function startLiveDataRefreshTimer() {
     GUI.timeout_add('data_refresh', update_live_status, 100);
 }
 
-async function update_live_status() {
+async function getStatus() {
```

Comment:
There is no awai, no reason to mark it as async
```suggestion
function getStatus() {
```